src/compiler/transformer.ts

```diff
@@ -116,6 +117,7 @@ namespace ts {
             transformers.push(transformJsx);
         }
 
+        transformers.push(transformESNext);

```

Comment:
I hadn't planned on it, But it's a good idea. I suppose people may want Babel to manage the emit, for example.